flask/datahandle.py

```diff
@@ -5,6 +5,53 @@
 
 ###Helper functions
 
+def formatRequisites(text, requisites):
+    """If any item requisites specified, adds them to response text data for more holistic response.
+
+    :param text: The response text data to be formatted
+    :type texta: string
+    :param requisites: Contains information food item must comply with (traits, allergens, etc)
+    :type requisites: dict
+    """
+    traitsText = ''
+    allergensText = ''
+
+    #If traits specified, extract into a string
+    for trait in requisites['trait']:
+        if traitsText:
+            traitsText += ', '
+        if trait == 'mhealthy':
+            trait = 'healthy'
+        traitsText += trait
+    traitsText = formatPlural(traitsText.rstrip(', '))
+
+    #If allergens specified, extract into a string
+    for allergen in requisites['allergens']:
+        if allergensText:
+            allergensText += ', '
+        if allergen == 'sesame-seed':
```

Comment:
Do you think it'd be worth reading these mapped values from a json file? Are these likely to change?

Reply:
How are you thinking of implementing that? With what I'm imagining it seems like it'd be easier to just stick with this since it's only 6 lines with no extra file to incorporate.

Reply:
Yeah, if it's just going to be these it's probably fine in here. I was thinking we'd have a key value like this.

This could also just be done in python too I guess to make this easier to deal with in the future.

JSON
```
{
  "traits": {
    "mhealthy": "healthy"
  },
  "allergens": {
    "sesame-seed":"seasme seeds",
    "tree-nuts":"tree nuts",
    "wheat_barley_rye":"wheat or barley or rye",
  },
}
```

PYTHON
```
fdict = {'traits': {'mhealthy': 'healthy'}, 'allergens': {'sesame-seed': 'sesame seeds', 'tree-nuts': 'tree nuts', 'wheat_barley_rye': 'wheat or barley or rye'}}
```

Then just do the replacement from the dict instead of the if. (These would be the variables rather than the strings of course)
`fdict["allergen"].get("mhealthy", "mhealthy")`

Reply:
Also if you just did the replacement "in-place" then you could maybe do it like this?

```
for i, trait in enumerate(requisites['trait']):
   requisites['trait'][i] = fdict["trait"].get(trait, trait)
```

And at the end . . . 
`", ".join(requisites['trait'])`

Reply:
Fixed :)